README.rst

```diff
@@ -36,7 +36,7 @@ To set up and run ``rp-sidekick`` locally, do the following::
     $ virtualenv ve
     $ source ve/bin/activate
     $ pip install -e .
```

Comment:
I think if we're changing this we might as well remove all the commands we don't need.

With uv we don't need a virtualenv, it takes care of it, we only need `uv sync`